app/src/main/java/org/p2p/wallet/infrastructure/network/provider/SendModeProvider.kt

```diff
@@ -0,0 +1,53 @@
+package org.p2p.wallet.infrastructure.network.provider
+
+import com.google.gson.annotations.SerializedName
+import org.p2p.core.utils.Constants.USD_READABLE_SYMBOL
+import org.p2p.wallet.infrastructure.security.SecureStorageContract
+import org.p2p.wallet.infrastructure.security.SecureStorageContract.Key
+import org.p2p.wallet.send.model.CurrencyMode
+import org.p2p.wallet.utils.emptyString
+import timber.log.Timber
+
+private const val TAG = "SendModeProvider"
+
+private val EMPTY_TOKEN = CurrencyMode.Token(emptyString(), 2)
```

Comment:
EMPTY_TOKEN_MODE and named args

Reply:
it is not a mode
it is just a Token for temp usage